pages/Getting Started/Installation.md

```diff
@@ -415,11 +415,6 @@ export CC=gcc CXX=g++ LDFLAGS="-static-libstdc++ -static-libgcc"
 
 refer to the Ubuntu tab above
 
-Please note that Hyprland uses the C++23 standard, so both your compiler and
```

Comment:
"old" should be changed to C++26 and gcc 14 and clang 18 respectively

Reply:
That is already in the "Manual (Manual Build)" section and is duplicate information.
https://github.com/hyprwm/hyprland-wiki/blob/aebff60c8f015cb86b5a392f115effd3454bfc7a/pages/Getting%20Started/Installation.md?plain=1#L377-L378

So, I deleted it.
Should I add this to Ubuntu section too?

Reply:
ah, thats fine ye